codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/EntityServiceSourceGenerator.scala

```diff
@@ -316,7 +482,7 @@ object EntityServiceSourceGenerator {
                                       className: String): Option[String] = {
     entity match {
       case eventSourcedEntity: ModelBuilder.EventSourcedEntity =>
-        None
+        Some(eventSourcedEntityProvider(service, eventSourcedEntity, packageName, className))
```

Comment:
I made it an option (here and above) because we didn't have a provider and a handler for ES. 

We can remove the Option now.

Reply:
ok, I'll remove that in next round